CONTRIBUTING.md

```diff
@@ -38,6 +38,13 @@ If you want to get started in the project, a good idea is to check issues labele
 You will need to fork the Rally repository and clone it to your local machine. See 
 the [Github help page](https://help.github.com/articles/fork-a-repo) for help.
 
+### Importing the project into IntelliJ IDEA
+
+Rally builds using virtualenv. When importing into IntelliJ you will need to define an appropriate Python SDK, which is provided by virtualenv.
+For more details on defining a Python SDK in IntelliJ please refer to their documentation. We recommend using the Python SDK that `make prereq` creates.
```

Comment:
Maybe we can even me more specific and just say add an "Existing environment" in the `Virtualenv Environment` dialog using `$RALLY_PATH/.venv/bin/python3` after `make prereq && make install` has been run. This should result in something like:

![image](https://user-images.githubusercontent.com/1754575/81040328-d5792d00-8eb3-11ea-91cb-e00901a7b061.png)

Reply:
I noticed the existing ES teams info did not include pictures, and I presume its because 1) we need a new folder to store the single pic and 2) that pic is guaranteed to change when intellij shakes up the UI every release. This is why I went with no pics, but I can definitely update the wording a bit.

Reply:
Sounds reasonable to me then.